.github/workflows/lint.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name: Linters
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    branches: ['main']
+    types: ['opened', 'reopened', 'synchronize']
+  workflow_dispatch:
+jobs:
+  yamllint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install yamllint
```

Comment:
This is not needed as the yamllint actions already installs yamllint.

Reply:
You're correct, and actually it seems `yamllint` is installed by default in the main images used in github actions, so even without the 3rd party action it would have worked (supposedly).

Reply:
fixed